### PR TITLE
[DOCFIX] Update the presto docs

### PR DESCRIPTION
**Pull request by gpang**

### CODE REVIEW

**Comment by gpang** [comment]
alluxio-bot, test this please

File: docs/en/compute/Presto.md [hunk 1]
```diff
@@ -20,6 +20,22 @@ latency especially when data is remote or network is slow or congested.
 * Table of Contents
 {:toc}
 
+## Using Presto with the Alluxio Catalog Service
+Currently, there are 2 ways to enable Presto to interact with Alluxio:
+* Presto interacts with the [Alluxio Catalog Service]({{ '/en/core-services/Catalog.html' | relativize_url }})
+* Presto interacts with the Hive Metastore (updated with Alluxio locations)
```
Comment (ZacBlanco) [inline docs/en/compute/Presto.md]: I think that "updated with Alluxio locations" is a little vague.

Suggestion "with table definitions updated to use Alluxio paths"?

File: docs/en/compute/Presto.md [hunk 2]
```diff
@@ -20,6 +20,22 @@ latency especially when data is remote or network is slow or congested.
...
+The primary benefits for using Presto with the Alluxio Catalog Service are simpler deployments
+of Alluxio with Presto (no modifications to the Hive Metastore), and enabling schema-aware
+optimizations (transformations like coalescing and file conversions).
+However, currently, the catalog service is limited to read-only workloads.
+
+For more details and instructions on how to use the Alluxio Catalog Service with Presto, please
+visit the [Alluxio Catalog Service documentation]({{ '/en/core-services/Catalog.html' | relativize_url }}).
+
+The rest of this page discusses the alternative approach of Presto interacting with the
+Hive Metastore with updated with Alluxio locations.
```
Comment (ZacBlanco) [inline docs/en/compute/Presto.md]: I think this can simply be left at "...interacting with the Hive Metastore."

File: docs/en/compute/Presto.md [hunk 3]
```diff
@@ -35,9 +51,10 @@ This guide is tested with `presto-315`.
 
 ### Configure Presto to connect to Hive Metastore
 
-Presto gets the database and table metadata information, as well as
-the file system location of table data from Hive Metastore.
-Edit the Presto configuration `${PRESTO_HOME}/etc/catalog/hive.properties`:
+Presto gets the database and table metadata information (including file system locations)from
```
Comment (ZacBlanco) [inline docs/en/compute/Presto.md]: space after `)`

File: docs/en/compute/Presto.md [hunk 4]
```diff
@@ -35,9 +51,10 @@ This guide is tested with `presto-315`.
...
+Hive Metastore, via the presto-hive-connector.
```
Comment (ZacBlanco) [inline docs/en/compute/Presto.md]: `the Hive Metastore`?

File: docs/en/compute/Presto.md [hunk 5]
```diff
@@ -35,9 +51,10 @@ This guide is tested with `presto-315`.
...
+Here is a example Presto configuration file `${PRESTO_HOME}/etc/catalog/hive.properties`,
+for a catalog using the hive-connector.
```
Comment (ZacBlanco) [inline docs/en/compute/Presto.md]: Should we use ``` `hive=connector` ```? or `Hive connector`?

File: docs/en/compute/Presto.md (hunk 1 above)
Comment (gpang) [reply docs/en/compute/Presto.md]: done

File: docs/en/compute/Presto.md (hunk 2 above)
Comment (gpang) [reply docs/en/compute/Presto.md]: done

File: docs/en/compute/Presto.md (hunk 3 above)
Comment (gpang) [reply docs/en/compute/Presto.md]: done

File: docs/en/compute/Presto.md (hunk 4 above)
Comment (gpang) [reply docs/en/compute/Presto.md]: done

File: docs/en/compute/Presto.md (hunk 5 above)
Comment (gpang) [reply docs/en/compute/Presto.md]: Using `Hive connector` now.

**Comment by gpang** [comment]
alluxio-bot, merge this please

**Comment by gpang** [comment]
alluxio-bot, cherry-pick this to branch-2.2 please

**Pull request merged**